README.md

```diff
@@ -1,15 +1,17 @@
-# polaris-ec2native-ondemand
+## :blue_book: Documentation
 
-## :hammer: Installation
+Export the following environment variables: 
+* `POLARIS_SUBDOMAIN`
+* `POLARIS_USERNAME`
+* `POLARIS_PASSWORD`
 
-## :mag: Example
+The values of `POLARIS_SUBDOMAIN`, `USERNAME`, and `PASSWORD` can also be assigned in *polaris-ec2native-ondemand.py* if you are retrieving from a secret store programatically somehow.
 
-## :blue_book: Documentation
+Ensure you set `snapcount` equal to the number of snapshots you want to retain, and enable or disable logging by setting `logging` equal to True or False in *polaris-ec2native-ondemand.py*
 
-Here are some resources to get you started! If you find any challenges from this project are not properly documented or are unclear, please raise an issueand let us know! This is a fun, safe environment - don't worry if you're a GitHub newbie! :heart:
+Once configured, run this script using Python 3.6+ from an EC2 instance being protected by Polaris. Each time it is run, it will take an on demand snapshot of this instnace and expire (starting with the oldest) any on demand snapshots this instance in excess of `snapcount`. 
```

Comment:
`instnace` typo

Reply:
`on demand snapshot of this instnace and expire`

Reply:
I would update the "How tested" section with a slightly more detailed explanation. Public reviewers may not know this test environment.

Reply:
fixed typo and added more detail as requested